### PR TITLE
Fixing https://wso2.org/jira/browse/APPCLOUD-249

### PR DESCRIPTION
**Pull request by manjulaRathnayaka**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/site/blocks/application/block.jag [hunk 1]
```diff
@@ -49,6 +49,14 @@ jagg.block("application", {
         var cloudSpecificApplicationRepresentation = jagg.module("manager").getCloudTypeProperties().cloudSpecificApplicationRepresentation;
         var applications = applicationModule.getAllApplications();
 
+        var requestNewAppTypeSubjectMessage = 'Requesting a new runtime for application type: '+inputs.appTypeName+' by tenant: ' + modManager.getTenantDomain();
+        var requestNewAppTypeURL = modManager.getPropertyValue(modManager.getCloudTypeProperties().properties.supportUrl) +
+                '&subject=' + requestNewAppTypeSubjectMessage;
+
+        var requestMoreAppsSubjectMessage = 'Requesting to run more than '+maxAppCount+' applications for tenant: ' + modManager.getTenantDomain();
+        var requestMoreAppseURL = modManager.getPropertyValue(modManager.getCloudTypeProperties().properties.supportUrl) +
+                '&subject=' + requestMoreAppsSubjectMessage;
```
Comment (amalkasubasinghe) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/block.jag]: word application should be project or integration cloud in both messages

File: modules/jaggeryapps/appmgt/src/site/blocks/apptype/select/block.jag [hunk 2]
```diff
@@ -36,8 +36,13 @@ jagg.block("apptype/select", {
             }
         }
 
+        var requestNewAppTypeSubjectMessage = 'Requesting a new application type by tenant: ' + manager.getTenantDomain();
```
Comment (amalkasubasinghe) [inline modules/jaggeryapps/appmgt/src/site/blocks/apptype/select/block.jag]: word application should be project or integration cloud

File: modules/jaggeryapps/appmgt/src/site/blocks/home/block.jag [hunk 3]
```diff
@@ -57,6 +57,10 @@ jagg.block("home", {
         var applicationCount = applicationModule.getRunningApplicationVersionCount();
         var cloudSpecificApplicationRepresentation = jagg.module("manager").getCloudTypeProperties().cloudSpecificApplicationRepresentation;
 
+        var requestNewAppTypeSubjectMessage = 'Requesting to keep application: '+application.applicationName+' running for an extended period of time in tenant: ' + managerModule.getTenantDomain();
+        var requestNewAppTypeURL = managerModule.getPropertyValue(managerModule.getCloudTypeProperties().properties.supportUrl) +
```
Comment (amalkasubasinghe) [inline modules/jaggeryapps/appmgt/src/site/blocks/home/block.jag]: word application should be project or integration cloud

**Comment by amalkasubasinghe** [comment]
word application should be project or integration cloud - check all messages

File: modules/jaggeryapps/appmgt/src/site/blocks/application/block.jag [hunk 4]
```diff
@@ -62,7 +70,9 @@ jagg.block("application", {
             "defaultVersion" : defaultVersion,
             "appTypeCount" : appTypeCount,
             "cloudSpecificApplicationRepresentation" : cloudSpecificApplicationRepresentation,
-            "applications" : applications
+            "applications" : applications,
+            "requestNewAppTypeURL" : requestNewAppTypeURL,
+            "requestMoreAppseURL" : requestMoreAppseURL
```
Comment (mcvidanagama) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/block.jag]: requestMoreAppseURL correct spelling

**Pull request merged**